fiftyone/zoo/models/manifest-torch.json

```diff
@@ -1742,31 +1742,28 @@
         },
         {
             "base_name": "complex-yolo-v3-torch",
-            "base_filename": "complex_yolov3",
+            "base_filename": "yolov3_ckpt_epoch-298.pth",
             "version": null,
             "description": "Complex extension of YOLO object detection model <https://arxiv.org/abs/1803.06199>`_ ",
             "source": "https://github.com/ghimiredhikura/Complex-YOLOv3",
             "size_bytes": 353976522,
             "manager": {
                 "type": "fiftyone.core.models.ModelManager",
                 "config": {
-                    "url": "https://github.com/ghimiredhikura/Complex-YOLOv3/blob/master/config/complex_yolov3.cfg"
+                    "google_drive_id": "1e7PCqeV3tS68KtBIUbX34Uhy81XnsYZV"
```

Comment:
By convention, `manager.config` is used to download the actual model weights.

Since we need to download an additional file in this case, which the existing infrastructure unfortunately doesn't support, we do still need to include `config_url` and `config_base_fiename` in the `default_deployment_config_dict.config`.

Reply:
For context, there are methods like `fiftyone.zoo.models. is_zoo_model_downloaded ()` which rely on using `manager.config` to download the model in order to correctly report whether the model is downloaded or not.

Ideally `is_zoo_model_downloaded() == True` would guarantee that no internet connection is required to use the model. But, until we get there, it's nice to let the builtin functionality handle the big part (model) and handle the small part (cfg) separately.